libraries/core_libs/consensus/include/vote_manager/vote_manager.hpp

```diff
@@ -298,13 +351,18 @@ class VoteManager {
   mutable boost::shared_mutex unverified_votes_access_;
   mutable boost::shared_mutex verified_votes_access_;
 
+  const size_t pbft_committee_size_;
   const addr_t node_addr_;
 
   std::shared_ptr<DbStorage> db_;
+  std::shared_ptr<PbftChain> pbft_chain_;
   std::shared_ptr<FinalChain> final_chain_;
   std::shared_ptr<NextVotesManager> next_votes_manager_;
   std::weak_ptr<Network> network_;
 
+  std::pair<blk_hash_t, std::unordered_map<vote_hash_t, std::shared_ptr<Vote>>> reward_votes_;
```

Comment:
This structure is kind of confusing, maybe it would be better instead of pair to have two separate variables like this:
blk_hash_t reward_votes_pbft_block_hash_;
std::unordered_map<vote_hash_t, std::shared_ptr<Vote>> reward_votes_;

Reply:
Fixed